Adicionado blogpost do Fillipe

vector-b opened the pull request:
Adicionado blogpost do Fillipe.
Não foi possível fazer teste da extensão para LaTeX 
Imagem de Blogpost de uso gratuito, disponível [Aqui](https://www.pexels.com/pt-br/foto/grafico-azul-e-amarelo-no-monitor-do-mercado-de-acoes-159888/)

vector-b committed:
Adicionado blogpost do Fillipe

vector-b committed:
tabelas centralizadas

vector-b committed:
tabelas centralizadas

abelsiqueira reviewed (changes requested):
@fillipepierin Parei ali no começo já com o seguinte review geral: verifique as equações. Esta versão da branch está com as equações funcionando.

vector-b committed:
Update _posts/2021-03-29-otimizacao-de-carteira-acoes.md

Co-authored-by: Abel Siqueira <nepper271@gmail.com>

vector-b committed:
Update _posts/2021-03-29-otimizacao-de-carteira-acoes.md

Co-authored-by: Abel Siqueira <nepper271@gmail.com>

vector-b committed:
Update _posts/2021-03-29-otimizacao-de-carteira-acoes.md

Co-authored-by: Abel Siqueira <nepper271@gmail.com>

vector-b committed:
Update _posts/2021-03-29-otimizacao-de-carteira-acoes.md

Co-authored-by: Abel Siqueira <nepper271@gmail.com>

Commit:
update blog

Commit:
update

Commit:
update nas formulas

Commit:
update nas formulas

Commit:
Correções de tabela

fillipepierin committed:
Create 2021-03-29-otimizacao-de-carteira-acoes0.md

fillipepierin committed:
Update 2021-03-29-otimizacao-de-carteira-acoes0.md

Tentando atualizar os comandos:
<script type="text/x-mathjax-config"> MathJax.Hub.Config({ tex2jax: { inlineMath: [['$$','$$'], ['$','$'], ['\\(','\\)']], processEscapes: true}, jax: ["input/TeX","input/MathML","input/AsciiMath","output/CommonHTML"], extensions: ["tex2jax.js","mml2jax.js","asciimath2jax.js","MathMenu.js","MathZoom.js","AssistiveMML.js", "[Contrib]/a11y/accessibility-menu.js"], TeX: { extensions: ["AMSmath.js","AMSsymbols.js","noErrors.js","noUndefined.js"], equationNumbers: { autoNumber: "AMS" } } }); </script>

Arrumando para ver se dá para exibir as equações: inlineMath: [['$$','$$'], ['$','$'], ['\\(','\\)']].
